manifests/git_repo.pp

```diff
@@ -0,0 +1,28 @@
+# Setup and configure a git repo
+define katello_devel::git_repo(
+  $source,
+  $github_username = undef,
+) {
+
+  validate_string($source)
+
+  $sources = {"${$katello_devel::upstream_remote_name}" => "https://github.com/${source}.git"}
```

Comment:
The array key here looks weird to me. Why is the string needed? Maybe this should also be a parameter to this define (and default to ```$katello_devel::upstream_remote_name```)?